docs/sources/reference/commandline/cli.md

```diff
@@ -799,7 +799,7 @@ Creates a new container.
       -p, --publish=[]           Publish a container's port(s) to the host
       --privileged=false         Give extended privileges to this container
       --read-only=false          Mount the container's root filesystem as read only
-      --restart=""               Restart policy to apply when a container exits
+      --restart="no"             Restart policy to apply when a container exits
```

Comment:
To be consistent:

Restart policy to apply when a container exits (no, on-failure[:max-retry], always)